apps/spruce/src/gql/client/cache.ts

```diff
@@ -8,6 +8,9 @@ export const cache = new InMemoryCache({
   typePolicies: {
     Query: {
       fields: {
+        adminEvents: {
+          keyArgs: [],
```

Comment:
Does it work if you set:
```suggestion
          keyArgs: false,
```

this means the query has no key fields to track, which seems to be the case here